src/main/webapp/dev/js/common/feedbackResponseComments.es6

```diff
@@ -5,10 +5,26 @@ import { StatusType } from './const.es6';
 import { destroyEditor, richTextEditorBuilder } from './richTextEditor.es6';
 import { toggleChevron } from './ui.es6';
 
+const initialVisibilityOptions = new Map();
+
 function isInCommentsPage() {
     return $(window.location).attr('href').indexOf('instructorCommentsPage') !== -1;
 }
 
+function setInitialVisibilityOfCheckboxes(id, element) {
+    const tableInForm = element.find('table').first();
+    const checkboxesInForm = tableInForm.find('tr').find('input.visibilityCheckbox');
+    const valuesOfCheckbox = [];
```

Comment:
Consider using an object/`Map` to explicitly associate the checkboxes and their values, instead of having an implicit association by array indices.